### ISSUE TITLE
Is this repo still being maintained?

### ISSUE THREAD

**Issue by IzhakJakov** [tomasiser/vim-code-dark#94]
Look like no commits have been made in a while there are a few issues/PRs that have not been addressed.

**Comment by tomasiser** [comment]
Hi @IzhakJakov,

Thanks for your question!

This repository was started more than 5 years ago and I would say that over the years, all of the contributors helped to reach quite a steady state where this colour scheme is working very well for the majority of users. There are always some edge cases and most of the current feature requests and PRs are mostly about adjusting colours a bit here and there so that they look a bit better in some specific settings. The problem of these is that in order to implement them, one needs to test the scenario in that specific settings, and I do not have a capacity to do that. Hence I rely on the community to submit PRs that I try to review and merge once in a while (the last three merged PRs were in December 2021, and before in April 2021).

Sometimes, users suggest changes that are very subtle and subjective, e.g., there are some suggestions to change some colour tones or the intensity of backgrounds, and these are especially tough to decide on, because if I merge such a PR, it suddenly affects all the users who have been using the colour scheme for years already (there are more than 700 stars on this repo in total, and more than 100 people clone this repo every day! thank you everyone who is using this colour scheme!).

So to summarize, this repository is in a rather maintenance mode (it is a colour scheme, so it is not really expected to change every day). I mostly rely on the community to give me feedback and offer changes and PRs, and I try to merge them once in a while!

**Comment by IzhakJakov** [comment]
OK thank you for clarifying in such detail.  I guess I should have just been patient :)

Here is another [fix some edge case](https://github.com/tomasiser/vim-code-dark/pull/96).

**Comment by IzhakJakov** [comment]
Hi @tomasiser
Have you seen the [fix for some other edge case](https://github.com/tomasiser/vim-code-dark/pull/96)?

**Comment by tomasiser** [comment]
Hi everyone again, after a long time, I managed to resolve all pull requests in this repo. Thanks to many contributors, the color scheme now looks more complete than it did ever before! I will continue to make sure that all PRs get resolved as soon as possible, although I can never promise anything as I am very busy at work. Once again, thanks! I will now close this issue.
